webpack/assets/javascripts/react_app/components/common/Loader/Loader.test.js

```diff
@@ -2,10 +2,10 @@ import toJson from 'enzyme-to-json';
 import { shallow, mount } from 'enzyme';
 import React from 'react';
 
-import { STATUS } from '../../constants';
-import Loader from './Loader';
+import { STATUS } from '../../../constants';
+import Loader from './index';
```

Comment:
nitpick: ```import Loader from './'``` also works

Reply:
tbh, i'm not sure what i prefer, i think `./index` is more reasonable to understand.